### ISSUE TITLE
After installing library I am getting a gradle error

### ISSUE THREAD

**Issue by trobrock** [bugsnag/bugsnag-react-native#7]
Error is related to min-sdk versions and I believe I've correctly tried the use tools override library suggestion:

```
Error:Execution failed for task ':bugsnag-react-native:processDebugAndroidTestManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 4 cannot be smaller than version 16 declared in library [com.facebook.react:react-native:0.33.1] /Users/trobrock/Sites/Frankenstein/node_modules/bugsnag-react-native/android/build/intermediates/exploded-aar/com.facebook.react/react-native/0.33.1/AndroidManifest.xml
  	Suggestion: use tools:overrideLibrary="com.facebook.react" to force usage
```

**Comment by trobrock** [comment]
Clearing my build cache and android/build folder seemed to fix this.

**Comment by trobrock** [comment]
Reopening, because this is persistent. Seems like clearing cache helps for a bit, but then this very quickly comes back.

**Comment by kattrali** [comment]
Apologies, someone I missed the notification for this issue. It has been addressed in v1.0.2. Let me know if this helps, @trobrock.
